components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/logging/APILogMessageHandler.java

```diff
@@ -0,0 +1,150 @@
+/*
+ *  Copyright WSO2 Inc.
```

Comment:
Do we need licence year. Seems we need to have some changes to this licence text as follow.

/*
*  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
*
*  WSO2 Inc. licenses this file to you under the Apache License,
*  Version 2.0 (the "License"); you may not use this file except
*  in compliance with the License.
*  You may obtain a copy of the License at